pkg/v1/random/image.go

```diff
@@ -141,3 +124,32 @@ func (i *image) LayerByDiffID(diffID v1.Hash) (partial.UncompressedLayer, error)
 	}
 	return l, nil
 }
+
+// Layer returns a layer with pseudo-randomly generated content.
+func Layer(byteSize int64, fileName string, mt types.MediaType) (v1.Layer, error) {
```

Comment:
Seems weird to have `random.Layer` take a filename. Can we just generate a random filename too?

If you think it should stay, at least document why it's needed in the doc comment.